Vinyl/Turntable.swift

```diff
@@ -32,31 +34,55 @@ public final class Turntable: NSURLSession {
             operationQueue = NSOperationQueue()
             operationQueue.maxConcurrentOperationCount = 1
         }
+        
+        if configuration.recodingEnabled {
+            recorder = Recorder(wax: Wax(tracks: []), recordingPath: configuration.recordingPath)
+            recordingSession = urlSession ?? NSURLSession.sharedSession()
+        }
```

Comment:
If you are checking this `urlSession ?? NSURLSession.sharedSession()`, its preferable to use:

```swift
+    public init(configuration: TurntableConfiguration, delegateQueue: NSOperationQueue? = nil, urlSession: NSURLSession = NSURLSession.sharedSession()) {
```

and make `recordingSession ` non optional and constant (`let`).